### PR TITLE
Wandb debugging

### PR DESCRIPTION
**Pull request by maxzvyagin**

### CODE REVIEW

**Review by KPHippe — APPROVED** [review]
Thanks for doing that!

File: gene_transformer/model.py [hunk 1]
```diff
@@ -173,10 +189,20 @@ def train(cfg: ModelSettings) -> None:
...
-            print("Using Weights and Biases for logging...")
+        node_rank = os.environ.get("NODE_RANK")
+        rank = os.environ.get("RANK")
+        local_rank = os.environ.get("LOCAL_RANK")
+        slurm_procid = os.environ.get("SLURM_PROCID")
+        jsm_namespace = os.environ.get("JSM_NAMESPACE_RANK")
+        wandb_active_env = os.environ.get("WANDB_ACTIVE")
+
+        print(
+            f"{rank=}, {local_rank=}, {slurm_procid=}, {jsm_namespace=}, {node_rank=}"
```
Comment (braceal) [inline gene_transformer/model.py:200]: We probably want to get rid of this print and os.environ calls to keep the script clean and avoid needing to specify default arguments to .get() that may confuse. 

The new logic looks great!

Comment (braceal) [reply gene_transformer/model.py:200]: The if statement can simply be:

```
if os.environ.get("WANDB_ACTIVE", 1):
```

Comment (braceal) [reply gene_transformer/model.py:200]: This can be added to line 191

**Comment by braceal** [comment]
**Updates**
- Fixing perlmutter slurm template
- Fixing multi-node wandb logging on polaris

**Pull request merged**